docs/pages/setup/reference/terraform-provider.mdx

```diff
@@ -58,7 +59,7 @@ provider "teleport" {
 ```
 
 
-## teleport_user
+## <a name="teleport_user"></a>teleport_user
```

Comment:
@gzigzigzeo there is no need for manual anchors, Next engine generates them automatically

Reply:
@klizhentas Did not know that. Removed.